apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,40 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I can mark a book as finished', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('c++');
+    await form.submit();
+
+    // Add Book to List
+    const items = await $$('[data-testing="book-item"]').first();
+    await items.$$('[data-testing="add-book"]').first().click();
```

Comment:
`first()`? Do we expect more than one? Should you be using `$` here?

Reply:
Made the appropriate change